service/gcsutils/gcstools/commoncli/common.go

```diff
@@ -79,6 +80,15 @@ func SetupLogging(args ...*string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("Invalid log params")
 	}
-	utils.SetLoggingOptions("verbose", *args[0])
+
+	logrus.AddHook(commonutils.NewStackHook([]logrus.Level{logrus.DebugLevel}))
```

Comment:
This doesn't seem like quite the right set of cases where we want the hook to fire. It seems like if I turn on debug logging (not sure how that's done in this case), then I want the hook to fire for all levels, not just the debug level.